### DISCUSSION TITLE
録音開始ボタンを「レコーディング」に変更

### DISCUSSION THREAD

**Discussion by CSReviser** [CSReviser/CaptureStream#11] [General]
録音開始ボタンを「レコーディング」に変更
デフォルトの拡張子を「mp3」から「m4a」に変更
※「mp3」を選択すれば、従来と同じ64kビットレートのmp3ファイルになります。

「今週放送分（試行）」ボタンを追加。
　※このボタンの状態は保存されません。毎回起動後にチェックが必要です。
　※講座名ファルダの中に「今週放送分」が作成され、今週放送分のファイルが格納されます。
　※試験的な実装です。再放送週や年末年始など動作しない可能性があります。
　※この機能の仕様は今後変更する可能性があります。

※ffmpegはバージョン5.0.1に対応しています。
※Mac版は環境がないので、動作未確認です。

<hr /><em>This discussion was created from the release <a href='https://github.com/CSReviser/CaptureStream/releases/tag/20220520'>録音開始ボタンを「レコーディング」に変更</a>.</em>

**Comment by DaddyJapan** [comment]
私のMacでは、問題なく動作しています。今週分も、金曜日の夜ですが、問題なく録音できています。今週分のフォルダーが作成されて、そこに入っています。

**Comment by butsuryo** [comment]
本バージョンWin版（CS-Mult-Windows-20220521.zip）をダウンロード後、展開しようとすると、
「ffmpeg.exe」のみ解凍エラーになります。

・解凍に使用しているツールはLhaplusです。
・Lhaplusでは「正常に解凍できません。CRCが一致しないか、出力先ファイルが使用中です」と警告が出ます。
・Win標準の解凍機能（右クリック＞プログラムから開く＞エクスプローラー を全コピー＆ペースト）でもエラーになりました。
・ひとつ前のバージョン「CS-Mult-Windows-20220518.zip」まではエラーが発生しません。

こちらの環境の問題でしょうか？zipの構造が何か変わったのでしょうか。
お手数ですが、ご確認いただけますと幸いです。

**Reply by butsuryo** [reply]
大変失礼いたしました、リポジトリを間違えました。
上記は「CSReviser/CS-Mult」へのコメントでした。

**Reply by CSReviser** [reply]
ご連絡ありがとうございました。
不具合を確認し、ファイル（CS-Mult-Windows-20220521.zip）を差し替えました。

**Reply by butsuryo** [reply]
ありがとうございます！エラーにならず解凍できることを確認いたしました。大変助かりました。
